inventory/group_vars/k8s-cluster.yml

```diff
@@ -132,3 +132,8 @@ efk_enabled: false
 
 # Helm deployment
 helm_enabled: false
+
+# dnsmasq
+dnsmask_upstream_dns_servers:
```

Comment:
Spelling: dnsmasq (not dnsmask)
Secondly, adding a fictitious nameserver by default is bad. It should be commented out.